spec/lib/serializable_spec.rb

```diff
@@ -81,6 +86,11 @@ class UserSerializer
       expect(result).to eq(user_data)
     end
 
+    it "Serialize nil, under [:data if resource is nil" do
```

Comment:
Missing closing bracket here 🤓 do you have merge access now?

Reply:
I do have merge access now, thanks!

Reply:
Sweet. Merge at will 👍